.gitmodules

```diff
@@ -23,4 +23,34 @@
 	url = https://github.com/nanopb/nanopb
 [submodule "icecap/icecap"]
 	path = icecap/icecap
-	url = https://gitlab.com/arm-research/security/icecap/icecap.git
+	url = https://gitlab.com/geky/icecap.git
```

Comment:
Can we keep this under an institutional/project workspace on Gitlab or Github, rather than a personal account?

Reply:
Yep, can do, currently this is in a PR over here: https://gitlab.com/arm-research/security/icecap/icecap/-/merge_requests/8, but needs some tweaks before it can be merged.

I have the submodule pointing at my fork right now just so that this PR can be used as is.